src/main/frontend/components/plugins.cljs

```diff
@@ -261,6 +261,12 @@
                  (js-invoke js/LSPluginCore (if disabled? "enable" "disable") id))
                true)]])
 
+(defn open-plugin-readme! 
```

Comment:
Suggest to name this function as
`get-open-plugin-readme-handler`

Reply:
fixed